draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2042,39 +2096,39 @@ Internet-Draft                 ACTN POI                      March 2022
 
    The MDSC translates the L2/L3 VPN SLA requirements into TE
    requirements (e.g., bandwidth, TE metric bounds, SRLG disjointness,
-   nodes/links/domains inclusion/exclusion) and find the SR-TE paths
-   that meet these TE requirements (see section 2.1.1).
+   nodes/links/domains inclusion/exclusion) and find the TE paths that
+   meet these TE requirements (see section 2.1.1).
 
 
 
 
-Peruzzini et al.      Expires September 7, 2022               [Page 38]
+Peruzzini et al.      Expires December 29, 2022               [Page 39]
 
-Internet-Draft                 ACTN POI                      March 2022
+Internet-Draft                 ACTN POI                       June 2022
 
 
    For example, considering the L3VPN in Figure 2, the MDSC finds that:
```

Comment:
General suggestion for this section: since the main content of the section is related to fig 2 why do not report again fig. 2 here: it very heavy to come back to figure 2 and reading the same time the text.

Reply:
Issue #96 has been created to address this comment in a future update: with markdown we can include the same figure multiple times within the document